src/components/generators/view-components/string.vue

```diff
@@ -43,9 +74,21 @@ export default {
   transition: transform 0.1s;
   box-shadow: 0 0 0 1px white;
 }
+
 .badge:hover {
   transform: scale(1.4);
 }
+
+.fa-clipboard {
+  margin-left: 5px;
```

Comment:
can you use a bootstrap class? ml-1 or ml-2? if not, try to use rems

Reply:
used ml-1